spec/rubocop/cop/rspec/factory_bot/create_list_spec.rb

```diff
@@ -65,105 +77,135 @@
 
-    include_examples 'autocorrect', bad_code, good_code
+      expect_correction(<<~RUBY)
+        create_list(:user, 3, :trait) { |user| create :account, user: user }
+      RUBY
+    end
 
-    bad_code = <<-RUBY
-      3.times do
-        create(:user, :trait) do |user|
-          create :account, user: user
-          create :profile, user: user
+    it 'flags usage of n.times with nested block arguments' do
+      expect_offense(<<~RUBY)
+        3.times do
+        ^^^^^^^ Prefer create_list.
+          create(:user, :trait) do |user|
+            create :account, user: user
+            create :profile, user: user
+          end
         end
-      end
-    RUBY
-
-    good_code = <<-RUBY
-      create_list(:user, 3, :trait) do |user|
-          create :account, user: user
-          create :profile, user: user
-      end
-    RUBY
+      RUBY
 
-    include_examples 'autocorrect', bad_code, good_code
+      expect_correction(<<~RUBY)
+        create_list(:user, 3, :trait) do |user|
+            create :account, user: user
+            create :profile, user: user
+        end
+      RUBY
+    end
   end
 
   context 'when EnforcedStyle is :n_times' do
     let(:enforced_style) { :n_times }
 
     it 'flags usage of create_list' do
-      expect_offense(<<-RUBY)
+      expect_offense(<<~RUBY)
         create_list :user, 3
         ^^^^^^^^^^^ Prefer 3.times.
       RUBY
+
+      expect_correction(<<~RUBY)
+        3.times { create :user }
+      RUBY
+    end
+
+    it 'flags usage of create_list with argument' do
+      expect_offense(<<~RUBY)
+        create_list(:user, 3, :trait)
+        ^^^^^^^^^^^ Prefer 3.times.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        3.times { create(:user, :trait) }
+      RUBY
+    end
+
+    it 'flags usage of create_list with keyword arguments' do
+      expect_offense(<<~RUBY)
+        create_list :user, 3, :trait, key: val
+        ^^^^^^^^^^^ Prefer 3.times.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        3.times { create :user, :trait, key: val }
+      RUBY
     end
 
     it 'flags usage of FactoryGirl.create_list' do
-      expect_offense(<<-RUBY)
-       FactoryGirl.create_list :user, 3
-                   ^^^^^^^^^^^ Prefer 3.times.
+      expect_offense(<<~RUBY)
+        FactoryGirl.create_list :user, 3
+                    ^^^^^^^^^^^ Prefer 3.times.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        3.times { FactoryGirl.create :user }
       RUBY
     end
 
     it 'flags usage of FactoryGirl.create_list with a block' do
-      expect_offense(<<-RUBY)
-       FactoryGirl.create_list(:user, 3) { |user| user.points = rand(1000) }
-                   ^^^^^^^^^^^ Prefer 3.times.
+      expect_offense(<<~RUBY)
+        FactoryGirl.create_list(:user, 3) { |user| user.points = rand(1000) }
+                    ^^^^^^^^^^^ Prefer 3.times.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        3.times { FactoryGirl.create(:user) } { |user| user.points = rand(1000) }
```

Comment:
Unrelated to this change, but it turns out this is a bad example - models are persisted before they are passed to the block, so the change to `points` won't be persisted. https://github.com/thoughtbot/factory_bot/issues/1444
We eventually may want to adjust this to `build`, `user.update` or add `user.save!`